library/std/src/sys/process/unix/fuchsia.rs

```diff
@@ -152,6 +152,11 @@ impl Process {
         Ok(())
     }
 
+    pub fn send_signal(&mut self) -> io::Result<()> {
+        // Fuchsia doesn't have a direct equivalent for signals
+        unimplemented!()
```

Comment:
Cc target maintainers @erickt @Nashenas88 just to double check there isn't something that makes sense here (nonblocking for this PR)

Reply:
Fuchsia and unsupported need a `_signal: i32` argument to match the other signatures

Reply:
Correct, fuchsia does not support signals. Thanks for checking!